stubs/gdb/gdb/__init__.pyi

```diff
@@ -663,18 +669,142 @@ class LineTable:
 
 # Breakpoints
 
+# The list of `case`s in the big `switch` block in py-breakpoint.c:bppy_init().
+_BreakpointType_NonWatchpoint: TypeAlias = Literal[BP_BREAKPOINT, BP_HARDWARE_BREAKPOINT, BP_CATCHPOINT]
+_BreakpointType_Watchpoint: TypeAlias = Literal[BP_WATCHPOINT]
+
+_WatchpointClass: TypeAlias = Literal[WP_READ, WP_WRITE, WP_ACCESS]
```

Comment:
Huh, mypy seems happy enough with this when checking my code that uses these hints, but in and pyright are upset about it when checking the stubs themselves.

> Variable not allowed in type expression (reportInvalidTypeForm)
> "Literal" must be None, a literal value (int, bool, str, or bytes), or an enum value (reportInvalidTypeForm)

Is there a proper way to write this? Should I turn `BP_*` and `WP_*` into enums? (They're enums in C, but the Python bindings have them as plain `int`s, not `enum.Enum`s).

Reply:
Are the integer values stable? If so, you could probably fill in the appropriate values in both the constants and the literal alias.

Reply:
I don't believe they're stable.